Landing-Page/index.html

```diff
@@ -26,41 +26,88 @@ <h1><span class="light">Site </span>Name goes here</h1>
 	<section id="showcase">
 		<div class="container">
 			<h1>Hi, Want to know where you belong</h1>
-			<p><i>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor magna aliqua"</p></i>
+			<p><i>“It does not matter how slowly you go as long as you do not stop.” — Confucius</p></i>
 			  <button type="submit" class="button_1">Dive In</button>
 			
 		</div>
 		
 	</section>
 
+
 	<section id="boxes">
 		<div class="container">
-			<h2>Word of Encouragement</h2>
+			<h1>Most Popular Viewed</h1>
 			<div class="box">
-				<h3>HTML5 Markup</h3>
-				<p><i>"Lorem ipsum dolor sit amet, consectetur
-			 adipiscing elit, sed do eiusmod tempor
-			  incididunt ut labore et dolore magna aliqua"</p></i>
+					<img src="img/software.jpg" alt="software-enginering-picture">
+				<h3>Software Engineer</h3>
+				<p>Software engineering combines the knowledge of maths, engineering and computer science to design, develop, test and evaluate software. There is a greater demand for good software engineers than ever before, thanks to a growing number of start-ups and increasing tech complexity. </p>
+				<button type="submit" class="button_2">Read More</button>
+				
 			</div>
 
 			<div class="box">
-				<h3>CSS3 Styling</h3>
-				<p><i>"Lorem ipsum dolor sit amet, consectetur
-			 adipiscing elit, sed do eiusmod tempor
-			  incididunt ut labore et dolore magna aliqua"</p></i>
+					<img src="img/computer-systems-engineering.jpg" width="computer-engineering-picture">
+				<h3>Computer systems Engineering</h3>
+				<p>Technology does not always operate in isolation. A systems engineer will combine a knowledge of maths, science and electronics with creative thinking to devise a workable, fluid computer system that is fast, efficient and productive in areas as diverse as robotics, home automation, biomedical devices and office workplaces. </p>
+				<button type="submit" class="button_2">Read More</button>
+				
 			</div>
 
-			<div class="box">	
-				<h3>Graphics Design</h3>
-				<p><i>"Lorem ipsum dolor sit amet, consectetur
-			 adipiscing elit, sed do eiusmod tempor
-			  incididunt ut labore et dolore magna aliqua"</p></i>
+			<div class="box">
+				<img src="img/web-developer.jpg" alt="web-developer-picture">
+				<h3> Web developer</h3>
+				<p>Web development is the building and maintenance of websites; it's the work that happens behind the scenes to make a website look great, work fast and perform well with a seamless user experience. Web developers, or ‘devs’, do this by using a variety of coding languages.  </p>
```

Comment:
This is duplicated. Are you making use of the content Augusta provided?

Reply:
changes Made